wp-content/themes/workingnyc/views/guides/single.twig

```diff
@@ -34,10 +34,6 @@
 
```

Comment:
Look at line 10: utility-nav.twig is used there. 
- [ ] You can change the props there to have the correct link and label.